newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -36,14 +54,52 @@ export default function SummaryPage() {
     }
   };
 
-  const sendResultEmails = () => _sendResultEmails(newdle.code);
+  const handleMailModalClose = useCallback(() => {
+    setMailModalOpen(false);
+  }, [setMailModalOpen]);
+
+  const handleMailModalConfirm = useCallback(() => {
+    setMailModalOpen(false);
+    _sendResultEmails(newdle.code);
+  }, [setMailModalOpen, newdle, _sendResultEmails]);
 
   if (!newdle) {
     return <Loader active />;
   }
 
   const mailSent = sendMailResponse !== null;
 
+  const recipientList = (list, color, icon, limit, limitSetter) => {
+    var sliced_list = list.slice(0, limit);
+    return (
+      <List>
+        {sliced_list.map((p, index) => {
+          return (
+            <List.Item key={p.id}>
+              <List.Icon color={color} name={icon} />
+              <List.Content>{p.name}</List.Content>
+            </List.Item>
+          );
+        })}
+        {limit < list.length && (
+          <List.Item>
+            <List.Icon color="black" name="ellipsis horizontal" />
+            <List.Content>
+              <a
+                href="#!"
```

Comment:
I think this adds an ugly `#!` to the URL - if you remove the href clicking the link should not change the URL at all.

Otherwise (in case you want a href for some reason) you could also use `evt.preventDefault()` in the onClick handler to prevent it from changing the URL.

Reply:
The problem here is that I think having a link in a List.Item looks nicer and is more intuitive for Users, but you can no longer have a <a /> without a valid href, and I couldn't find a nice way to format a Button as a link. Changed in this latest version to use a Button that you're already using. If you're happy with that we could leave it

Reply:
`<a>` should work fine without a href.. or is the styling different in that case?

Reply:
![Screenshot 2020-09-15 at 13 53 55](https://user-images.githubusercontent.com/6168071/93207217-f05d8700-f75a-11ea-85a3-9ac9c4f09da9.png)

Reply:
```
Line 56:9:  The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid
```

Reply:
ah that warning :/ i think going for the label like in the other place was a good choice :)